### ISSUE TITLE
The confusion on Results of NRMS

### ISSUE THREAD

**Issue by whonor** [jppol-ai/ebnerd-benchmark#18]
Hi there,

I reproduced the NRMS on the enberd_small dataset following this repository, splitting the training data into a training and validation set, considering the original validation set as the test set. The results are as follows:

<img width="1014" height="204" alt="Image" src="https://github.com/user-attachments/assets/1e480c30-8899-42c3-b1e7-002532c3dffe" />

The first column shows the original model performance on the MIND dataset. I think the results on the ebnerd_small dataset are too low, indicating random ranking.

I'm confused; is this correct, or is something wrong?

Thanks.

**Comment by johanneskruse** [comment]
Hi,

My apologies for the late response. First, thanks for your interest and for reproducing the results.

Yes, an AUC of ~0.5 is equivalent to random guessing. To best answer your question, what are you running to obtain the results? For example, are you running the notebooks or the command lines to reproduce them, and what dataset is being used for evaluation?

The results reported in the paper are based on:
```
python examples/reproducibility_scripts/ebnerd_nrms.py
  --datasplit ebnerd_small \
  --epochs 5 \
  --bs_train 32 \
  --bs_test 32 \
  --history_size 20 \
  --npratio 4 \
  --transformer_model_name FacebookAI/xlm-roberta-large \
  --max_title_length 30 \
  --head_num 20 \
  --head_dim 20 \
  --attention_hidden_dim 200 \
  --learning_rate 1e-4 \
  --dropout 0.20
```

We train on `large` and combine the training and validation splits, as described in the paper: [EB-NeRD: A Large-Scale Dataset for News Recommendation](https://dl.acm.org/doi/epdf/10.1145/3687151.3687152).
